.changeset/tidy-terms-act.md

```diff
@@ -0,0 +1,12 @@
+---
+'@primer/react-brand': minor
+---
+
+Replaced logical `end` values with physical `flex-end` across multiple components to resolve downstream compilation issues with older browserslist configs. This change removes automatic RTL support where `end` was previously used.
```

Comment:
```suggestion
Replaced `end` values with `flex-end` across multiple components to resolve downstream compilation issues with older browserslist configs.
```

I believe `flex-end` is still a logical value since flex axes are logical rather than physical. This change simply uses a flexbox-specific value instead of the broader `end` value, RTL should be working as before.